### DISCUSSION TITLE
My Blind Coding Journey

### DISCUSSION THREAD

**Discussion by heartofstoneclothing** [community/community#80956] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

# Hello Everyone :joy:
---
## Some Context 

- **The Idea** :bulb:
This is question I've been longing to ask but can't find the words to ask it, partly because I don't like the idea of being rejected when asking for help and also can't definitively express what it is I want to ask.  I purchased a domain name in the beginning of the year, I had picked up a pencil and started sketching about 15 years ago planned on using this domain to sell ~~clothing~~ coding based off my designs.
- **The Build-up** :1234:
The first few months of using the website builder were great, I enjoyed the whole building and design process, until I didn't. There was a lot more than I wanted to accomplish other than what the Website Builder would allow.

---
## Overview

- **The Complication** :heavy_multiplication_x: [^1]
Most of my life I only ever used Google, but my registrar was constantly pushing Microsoft 365 and how imperative it is that I get it. I did eventually and now I prefer Microsoft, but I don't quite understand it fully which has lead me here.
- **The Issue** :envelope:
I managed to get my email sorted out, allowing me at the time to use my new business email address with my registrar, Google, and Microsoft, oh and Outlook. I did something, I'm not sure exactly what, but now I can send e-mails from my business e-mail using my personal g-mail account. :man_shrugging:
- **The Unforeseen** :roll_eyes:
I got bored with the limitations of the ~~scrapbook~~ website builder and every time I needed to log in to my account a <span>popup</span> would appear asking if I wanted to use Github to log into my Microsoft account. I never knew I needed Github and only used it for authentication purposes for a long time before I actually did anything noteworthy. Every time I tried I encounter an error :exclamation: so I try again and lose another fragment of my repository. :brain::put_litter_in_its_place:

---
## Backlog

- **The Lesson** :memo:
I came across Microsoft's Learn documentation and was lured into their learning paths achieving to date 9 trophies :1st_place_medal: and 38 certificates :2nd_place_medal: I had the opportunity to be invited to their Cloud Migration Summit earlier in the year plus 2400hours at [freecodecamp](https://freecodecamp.org) :fire: earning 4 certificates  :3rd_place_medal: I struggle with this challenge though and understanding how to resolve the issue.
- **The _"know-how"_**
In trying to work my way out of this mess I have contacted Microsoft and Google Support, learned multiple new languages that look like heiroglyphics. `~/.:<!@#$%^[](0){->_\?''"*|&"` The computer has been reset which helped me understand what happened.
 

> It's kind of a long story, I messed up! I lack the knowledge right now to fix it. I logged onto git using my personal email at a time. The thought came, "hmm, I wonder what coding is like, I wonder how I can use it to drive myself mad for a year. Let's try things you don't know, then use it to turn everything you touch into a kaleidoscope of confusion and then build an entire project around it claiming it was for fun. :rofl:

### **The Mission**:syringe:
<details>
<summary>
<details>
if you choose to accept it
</details>
</summary>
<summary>
- Locate and extract my code from the other user's folders
</summary>
<summary>
- Undo the changes I've done to that User's profile 
</summary>
<summary>
- Gather resources and initialize project safely
</summary>
</details>

A Visual aid [^1]
| Reference | Account | Composition | Description | Business | Private | 
| --- | --- | --- | --- | --- | --- |
| 2.1. [^2.1.] | myPersonalAccount | 'myName' + '@' + 'gmail' + '.' + 'com' | main account | no | yes |
| 2.2. | myBusinessAccount | 'myName' + '@' + 'myBusiness' + '.' + 'com' | office use | yes| no |
| 2.3. | myOutlookAccount | 'myName' + 'myBusiness' + '@' + 'outlook' + '.' + 'com' | outdoor use | yes | yes |
| 2.4. | myPrivateAccount | 'string-of-numbers' + '@' + 'outlook' + '.' + 'com' | nsfw mode | no | yes | 

###### _note to self_ [^3]

[^1]: **The Plea** :pray: 
"Please could someone help me! 
[^2.1.]: 2.1. I have a Google account
my personal account ***`duh-duh-duh@gmail.com`***
which is the owner of and admin of
2.2. my business account ***`my-name@my-business.com`***
at my registrar and Microsoft but that same account gets recognized as
2.3. ***`'myName' + 'myBusiness' + '@outlook.com'`***
I'm not sure why it had to be done like this, but I distinctly remember it not being my intention and to protect my privacy my outlook account mentioned above also gets recognized as 
2.4. ***`'a-string-of-numbers' + '@outlook.com'`***"

[^3]: It was in creating footnotes you started fully grasping the possibilities of unix. :bulb: paint with keys, stop turning this into a project!

**Comment by ghostinhershell** [comment]
Hey @heartofstoneclothing, welcome to the Community and thanks so much for sharing your journey with us! 🚀
